taskworld/prescript
# add runto command

Add `runto` command to run until specific step.

Most of the time, when we setup prescript test with selenium, we wish to skip all the setup step. 

For example, we mostly ended up like this

Test A:

1. Create user
 1.1. Enter input username
 1.2. Enter password
 1.3. Confirm password
 1.4. Click on...
 1.5. .... etc
2. Confirm email
 2.1 Go to email website
 2.2 Login email address
 2.3 .....
3. .....
4. ....
5. Real test start here

Test B: 

1. Create user
 1.1. Enter input username
 1.2. Enter password
 1.3. Confirm password
 1.4. Click on...
 1.5. .... etc
2. Confirm email
 2.1 Go to email website
 2.2 Login email address
 2.3 .....
3. .....
4. ....
5. Real test start here

The `next` functionality have no benefit in these cases. We need command to immediately goto step 5 and then start debugging test from there
